DotNetHowToEncryptionUsingCMK/DotNetHowToEncryptionUsingCMK/Hotel.Methods.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace AzureSearch.SDKHowTo
+{
+    using System;
+    using System.Text;
+
+    public partial class Hotel
+    {
+        // This implementation of ToString() is only for the purposes of the sample console application.
+        // You can override ToString() in your own model class if you want, but you don't need to in order
+        // to use the Azure Search .NET SDK.
+        public override string ToString()
+        {
+            var builder = new StringBuilder();
```

Comment:
@mrcarter8 This uses the older non-complex Hotels schema. I'm actually ok with this as it keeps the sample small, and complex types isn't a necessary part of this scenario. What do you think?